### ISSUE TITLE
:pipe_ignore command

### ISSUE THREAD

**Issue by aral** [helix-editor/helix#3947]
Would be useful to have a `:pipe_ignore` command that’s the equivalent of `Alt-|` (pipe and ignore output).

e.g., I was playing around with supporting Ctrl+S for save in insert mode and the following works but, of course, I don’t want the output (nothing) replacing the current selection:

```
[keys.insert]
C-s = ["normal_mode", ":pipe sleep 0.1", ":write"]
```

So, the following would work (and update the dirty flag/indicator also):

```
[keys.insert]
C-s = ["normal_mode", ":pipe_ignore sleep 0.1", ":write"]
```

**Comment by the-mikedavis** [comment]
This might be better accomplished by https://github.com/helix-editor/helix/pull/3247. A pipe doesn't feel like the right tool since this use-case doesn't need stdin or stdout.

**Comment by aral** [comment]
Yeah, it’s a hack anyway :) Anything that delays it (to the next stack frame?)
